scripts/tools/silabs/ota/ota_multi_image_tool.py

```diff
@@ -163,6 +164,32 @@ def generate_app(args: object):
         return [OTA_APP_TLV_TEMP, args.app_input_file]
 
 
+def generate_wifi_image(args: object):
```

Comment:
The function is a copy paste of the generate app function outside of the tag change. 

Why do we need a specific tag to the M4 / TA image?
If we do need a specific tag, we should modify the function to take the tag as input instead.

Reply:
917 NCP TA alone upgrade and combined image upgrade needs separate tag based on that  wifi processor will be selected. In case of combined image (917 NCP) , both processors will be registered and processor will be changed once first TLV image is processed completed.